packages/shared-core/src/filters.ts

```diff
@@ -204,12 +235,29 @@ export const buildLuceneQuery = filter => {
   return query
 }
 
+const deepGet = (obj: { [x: string]: any }, key: string) => {
```

Comment:
There's already a `deepGet` method in **bbui** Helpers. Unless that's changed. Still it seems generic enough to be moved outside of the `filters.ts` file?

Reply:
This new package will be able to get all these use cases to write ts code shareable in both ts and js. I did not want to move many things now, but this was an easy one. Done